### PR TITLE
ha: refine integration_tests

### PR DESCRIPTION
**Pull request by lichunzhu**
### What problem does this PR solve? 
Refine integration tests for DM-HA.

### What is changed and how it works?
Update the mysql-worker start-up method.

**safe_mode, sequence_safe_mode, sequence_sharding, sharding** contain shard ddl tests and can't pass the integration_tests.

### Check List 

Tests 

 - Unit test
 - Integration test
 
Code changes

Side effects

Related changes

### CODE REVIEW

**Comment by lichunzhu** [comment]
@csuzhangxc @nolouch PTAL

File: tests/dmctl_advance/check_list/break_ddl_lock.sh [hunk 1]
```diff
@@ -14,20 +14,18 @@ function break_ddl_lock_without_worker() {
...
         "\`exec\` and \`skip\` can not specify both at the same time" 1
 }
 
 function break_ddl_lock_while_master_down() {
-    # worker address is only a placeholder in this test case, its value makes no sense
-    worker_addr="127.0.0.1:$WORKER1_PORT"
     run_dm_ctl $WORK_DIR "127.0.0.1:$MASTER_PORT" \
-        "break-ddl-lock test -w $worker_addr --exec" \
-        "can not break DDL lock (in workers \[$worker_addr\]):" 1
+        "break-ddl-lock test -w $MYSQL1_NAME --exec" \
```
Comment (csuzhangxc) [inline tests/dmctl_advance/check_list/break_ddl_lock.sh]: How about rename other `$worker_addr` under _tests/dmctl_basic_?

**Comment by csuzhangxc** [comment]
@lichunzhu `Integration Test relay_interrupt` failed

**Pull request merged**